TDAT: public void test get cause ( ) throws exception {  <NL>  / / construct a throwable with a root cause <NL> exception root cause = new exception ( ) ;  <NL>  <NL> throwable t = create caused exception ( " message " , root cause ) ;  <NL>  <NL> assert equals ( " root cause should be the throwable passed in on " +  <NL>  " construction " , root cause ,  <NL>  ( ( throwable extensions ) t ) . get cause ( ) ) ;  <NL>  }  COM: <s> tests the get cause method </s>